src/Microsoft.Extensions.Configuration.Azure.KeyVault/DefaultKeyVaultSecretManager.cs

```diff
@@ -14,7 +14,7 @@ public class DefaultKeyVaultSecretManager : IKeyVaultSecretManager
         /// <inheritdoc />
         public virtual string GetKey(Secret secret)
         {
-            return secret.SecretIdentifier.Name.Replace("__", ConfigurationPath.KeyDelimiter);
+            return secret.SecretIdentifier.Name.Replace("--", ConfigurationPath.KeyDelimiter);
```

Comment:
@glennc `--` used for any common configuration values?
Also, should this be documented somewhere?